instance/models/server.py

```diff
@@ -23,21 +23,19 @@
 # Imports #####################################################################
 
 import logging
-import novaclient
 import time
 
-from swampdragon.pubsub_providers.data_publisher import publish_data
-
+import novaclient
```

Comment:
@itsjeyd Good catch too. The change pylint introduced (requiring to import core libs before external dependencies) threw me a bit off balance and it took me a few tries to find a new ordering that was both happy with and that was consistent. I originally tried to keep `import ...` before `from ... import ...` for aesthetic, but that just adds too much constraints, and alphabetical ordering is more practical. So I switched in the middle the the changes, and it looks like I didn't properly re-apply it everywhere at the end. Just did a pass in https://github.com/open-craft/opencraft/pull/6/commits/1216ddaec678259f01de679b561031f246851f24 - see if that's better : )

For reference, the ordering is:

1. Core libs
2. External dependencies
3. Internal dependencies

Each of them ordered alphabetically, from the top-level module name, regardless of the import format.